src/scss/partials/_nav.scss

```diff
@@ -0,0 +1,104 @@
+$green: #9cdad7;
```

Comment:
All variables belong in the `_variables.scss` file.

Reply:
if I move this colour I am getting an error that is not defined.

Reply:
You need to `@import "variables";`

Reply:
done